internal/utils/process/checksumfile.go

```diff
@@ -8,9 +8,13 @@ import (
 
 // Creates checksum file name according to template <image name:tag>.<algorithm>
 // e.g nginx:lates.md5
-func CheckSumFile(procName, alg string) string {
+func CheckSumFile(procName, alg string) (string, error) {
```

Comment:
Naming? And may be skip this function at all?

Reply:
??